pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -74,6 +76,11 @@ stdenv.mkDerivation rec {
     cp "$desktopItem"/share/applications/* "$out/share/applications/"
   '';
 
+  preFixup = stdenv.lib.optionalString stdenv.isLinux ''
+    wrapProgram $out/bin/prusa-slicer \
+      --prefix LD_LIBRARY_PATH : "${systemd.lib}/lib"
```

Comment:
If you add systemd to the `buildInputs` and also set

```
NIX_LDFLAGS = "-ludev";
```

then you can force link `libudev` into `prusa-slicer` and make it also available to `dlopen`.
This generally causes less problems than `LD_LIBRARY_PATH` because the latter one is also inherited by child processes that might be linked against incompatible libraries.

Reply:
Changed it :)